services/queue/src/claimresolver.js

```diff
@@ -99,25 +103,18 @@ class ClaimResolver {
       }
     }));
 
-    // If there were no messages, back of for a bit.  This avoids pounding
-    // Azure repeatedly for empty queues, at the cost of some slight delay
-    // to finding new messages in those queues.
+    // If there were no messages, back of for a bit.
```

Comment:
I see it was there beforehand too, but here's a quick fix :)

```suggestion
    // If there were no messages, back off for a bit.
```